Makefile

```diff
@@ -58,42 +58,55 @@ build: .generate_v2_certs
 	@sed -i.bkp 's@/secc/g@/evcc/g@g' iso15118/evcc/Dockerfile
 	docker-compose build
 
+# Run using dev env vars
 dev:
 	# the dev file apply changes to the original compose file
 	docker-compose -f docker-compose.yml -f docker-compose.dev.yml up
 
+# Run using prod env vars
 run:
 	docker-compose -f docker-compose.yml -f docker-compose.prod.yml up
 
+# Update dependencies with poetry
 poetry-update:
 	poetry update
 
+# Install dependencies with poetry
 install-local:
 	poetry install
 
+# Use poetry virtual environment
 poetry-shell:
 	poetry shell
 
+# Run evcc with python
 run-evcc:
 	$(shell which python) iso15118/evcc/main.py
 
+# Run secc with python
 run-secc:
 	$(shell which python) iso15118/secc/main.py
 
+# Run mypy checks
 mypy:
 	mypy --config-file mypy.ini iso15118 tests
 
+# Reformat with isort and black
 reformat:
 	isort iso15118 tests && black --line-length=88 iso15118 tests
 
+# Run black checks
 black:
 	black --check --diff --line-length=88 iso15118 tests
 
+# Run flake8 checks
 flake8:
 	flake8 --config .flake8 iso15118 tests
 
-code-quality: reformat mypy black flake8
+# Run black, isort, mypy, & flake8
+code-quality: reformat mypy flake8
```

Comment:
Was black accidentally removed?

Reply:
`reformat` runs black so we were actually running it twice here